source/include/sigv4.h

```diff
@@ -414,13 +406,13 @@ SigV4Status_t SigV4_GenerateHTTPAuthorization( const SigV4Parameters_t * pParams
  * in length (excluding the null character), to comply with RFC 3339 and RFC
  * 5322 formats, respectively.
  * @param[in] dateLen The length of the pDate header value. Must be either
- * #SIGV4_EXPECTED_LEN_RFC_3339 or #SIGV4_EXPECTED_LEN_RFC_5322, for valid input
```

Comment:
Why are these links removed?

Reply:
Checks were showing linking errors for these macros/flags, so I had referenced other libraries that had similarly removed them since I was unable to find any standard on this.